Server/mods/deathmatch/logic/luadefs/CLuaAccountDefs.cpp

```diff
@@ -513,6 +514,12 @@ int CLuaAccountDefs::RemoveAccount(lua_State* luaVM)
     return 1;
 }
 
+bool CLuaAccountDefs::SetAccountSerial(std::string name, std::string serial) noexcept
```

Comment:
I think it would be better to use `CAccount` since the other functions use it as well
```suggestion
bool CLuaAccountDefs::SetAccountSerial(CAccount* account, std::string serial) noexcept
```

Reply:
adjusted 790ffa2